core/src/main/java/feign/Util.java

```diff
@@ -360,4 +352,33 @@ public static boolean isNotBlank(String value) {
   public static boolean isBlank(String value) {
     return value == null || value.isEmpty();
   }
+
+  /**
+   * Copy entire map of string collection.
+   *
+   * The copy is unmodified map of unmodified collections.
```

Comment:
Typo: should be _unmodifiable_ (those that you cannot modify), not _unmodified_ (those that you didn't modify)?